### DISCUSSION TITLE
i am trying to build native app with Shiny.BluetoothLE, MAUI for windows, andriod and ios but on windows its not showing output

### DISCUSSION THREAD

**Discussion by roysiki** [shinyorg/shiny#1345] [Q&A]
guys i hope you are doing fine and best of your health.. so i have used Maui in order to developed an cross platform app with Maui and it works fine with Android but when i compile code for windows machine the code build without any error but it doesn't show any ble devices around me and in the mean time if i run the same code on local emulator (on my android mobile ). it shows the devices around me so in short it works perfectly on my android device but when i compile for windows it doesnt work.. idk its either library issue or windows permission issue since i am new to MAUI .. and in log it just show one line idk it could be error or it could be because of that btw it says Exception thrown: 'System.NotImplementedException' in Shiny.BluetoothLE.dll . plus i have attached the screenshot . i hope you can give some valuable input thanks 
[
![error](https://github.com/shinyorg/shiny/assets/148053717/273891a9-4f4a-460d-b635-8fc92bdbf207)
](url)

**Comment by aritchie** [comment]
I don't support windows for pretty much anything in Shiny right now.

**Reply by roysiki** [reply]
ohhhh.. can you suggest me anything or any library which can work uwp as well as andriod and ios by using maui as i am new to this language.. or you could suggest me anything.. plus thanks for reply
